### ISSUE TITLE
Request: iCloud access

### ISSUE THREAD

**Issue by Rickard9** [opengisch/QField#5754]
It would be awesome to be able to access projects saved on iCloud from an iOS device

Best regards

**Comment by Rickard9** [comment]
sorry, this should be in Discussions

**Comment by m-kuhn** [comment]
Thanks for reporting this.
A better integration of import/export on ios would be a great enhancement.

**Comment by Rickard9** [comment]
Definitely, i am a new Qfields user and the only way I found is to import via the Qfield Cloud, witch works ok.

However if it would be possible to access iCloud to open projects and or move files from iCloud to the local folder would be sooo good, both for user experience and as an redundency for the Qfield cloud.
